### PR TITLE
Carpburster

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

Adds a unique suicide for the dehydrated carp plushie.

## Why It's Good For The Game

I guess it's funny?

## Changelog
:cl:
add: Dehydrated carp suicide.
/:cl:

### CODE REVIEW

File: code/game/objects/items/dehy_carp.dm [hunk 1]
```diff
@@ -47,3 +47,28 @@
...
+			return SHAME
+		if(H && !QDELETED(H))
+			H.spawn_gibs()
+			H.apply_damage(200, def_zone = BODY_ZONE_CHEST)
+			forceMove(get_turf(H)) //we move it back
+		icon = 'icons/mob/animal.dmi'
+		flick("carp_swell", src)
+		sleep(6) //let the animation play out
+	
+		if(src && !QDELETED(src))
```
Comment (MrDoomBringer) [inline code/game/objects/items/dehy_carp.dm]:
```suggestion
		if(!QDELETED(src))
```

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

**Comment by nemvar** [comment]
I just realized that this is vore content.

**Comment by MCterra10** [comment]
Quick, merge before orange man sees it

**Comment by FantasticFwoosh** [comment]
_Oh the Humanity!_

**Pull request merged**